hw/ip/otbn/doc/_index.md

```diff
@@ -123,10 +123,51 @@ CSRs can be accessed through dedicated instructions, `CSRRS` and `CSRRW`.
     <tr>
       <td>0x7C0</td>
       <td>RW</td>
+      <td>
```

Comment:
This table is a bit unwieldy, was mostly trying to add to what existed and matching that. We might consider a more compact layout.